pattern-library/fonts/edx-icons/edx-icons.scss

```diff
@@ -0,0 +1,110 @@
+$icon-linkedin-square: "\e600";
```

Comment:
This is spit out from Icomoon, but it's unused at the time. I was thinking it might be nice to have a Sass partial in the chance we wanted to do something... but I'm not sure I can see a purpose. Thoughts?